source/content/guides/wordpress-configurations/08-installing-updating-from-third-party-sources.md

```diff
@@ -0,0 +1,97 @@
+---
+title: WordPress Configurations Guide
+subtitle: Installing and Updating Plugins and Themes from Third-Party Sources
+description: Learn how to set up plugins and themes that are set up in external repositories.
+contributors: [jazzsequence, pwtyler, stevector]
+contenttype: [guide]
+innav: [false]
+categories: [config]
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [plugins]
+tags: [code, plugins]
+reviewed: "2024-10-16"
+permalink: docs/guides/wordpress-configurations/installing-updating-from-third-party-sources
+---
+
+This section provides guidance on how to manage plugins that use third-party sources beyond the WordPress plugin ressources. Such places include GitHub, Bitbucket, GitLab or self-hosted sources.
+
+<Alert title="Note" type="info" >
+All the guidance in this document is intended to be used in conjunction with Dev or Multidev environments on Pantheon where version-controlled files can be changed directly. [Pantheon's security practices lock down file permissions in the Test and Live environments](https://docs.pantheon.io/guides/filesystem/files-directories#write-access-on-environments).
+</Alert>
+
+## Using Git Updater to install and manage plugins (and themes) from Git repositories
+
+[Git Updater](https://github.com/afragen/git-updater) is a plugin that allows you to install and update plugins and themes from GitHub, Bitbucket, GitLab, or other self-hosted Git repositories. It provides both a mechanism that plugins can use to suggest updates from a Git-based repository as well as a user interface to install plugins from a Git repository that has not been previously configured to do so. Plugins and themes that use Git Updater can then be updated directly from the WordPress dashboard.
+
+0. Before you begin, make sure the Dev or Multidev environment in which you are working  is set to SFTP mode. That mode can be changed either from the Pantheon dashboard or by using the Terminus `connection:set` command.
+	```bash{promptUser: user}
+	terminus connection:set <site>.<env> sftp
+	```
+1. Download the latest zip file for the Git Updater plugin from the [Git Updater website](https://git-updater.com/) or use [WP-CLI through Terminus](https://docs.pantheon.io/guides/wp-cli) to install the plugin from its zip file in the GitHub repository releases page:
+
+	```bash{promptUser: user}
+	terminus wp <site>.<env> -- plugin install https://github.com/afragen/git-updater/releases/download/12.6.0/git-updater-12.6.0.zip --force
+	```
+
+	If you have [`jq`](https://jqlang.github.io/jq/) installed locally, you can get the latest version of Git Updater from the releases page (rather than a specific version) by using the following variation:
```

Comment:
@rachelwhitton can you gut check. While I don't like hard-coding a version number in the above prompt, I think bringing in `jq` and `$()` is too messy to document. I think I should delete this option.